docs/content/en/docs/implementing/day-2-operations/_index.md

```diff
@@ -0,0 +1,295 @@
+
+## Updating Workload Versions
+
+After a first successful deployment of your application with Keptn,
+you will eventually need to update the version of one or
+more workloads that are part of the application.
+This usually involves updating the image of a deployment
+and changing the configuration of a deployment.
+For example, using a different service account name for a pod
+could be seen as an update.
+Regardless of that, however, it is the user who decides what Keptn
+sees as a version bump in the application by setting the value of
+the `keptn.sh/version` or `app.kubernetes.io/version` labels/annotations
+in their workloads.
+
+When this changes, Keptn interprets a change as a new version
+and thus re-runs the pre- and post-tasks and evaluations for the application.
+
+If the version label/annotation does not change, Keptn does not consider
+a change of a workload configuration to be an update, and therefore no pre-
+and post-tasks/evaluations are executed because they have already been
+completed for the related `WorkloadVersion`.
+
+To illustrate the update of a workload,
+let's assume the following example, including
+a workload called `podtato-head-frontend` that includes a pre-task and
+a pre-evaluation.
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: podtato-kubectl
+  annotations:
+    keptn.sh/lifecycle-toolkit: "enabled"
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: podtato-head-frontend
+  namespace: podtato-kubectl
+spec:
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: podtato-head-frontend
+        app.kubernetes.io/part-of: podtato-head
+        app.kubernetes.io/version: 0.1.0
+        keptn.sh/pre-deployment-evaluations: my-evaluation
+        keptn.sh/pre-deployment-tasks: my-task
+    spec:
+      containers:
+        - name: podtato-head-frontend
+          image: podtato-head-frontend:a
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: podtato-head-hat
+  namespace: podtato-kubectl
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: podtato-head-hat
+        app.kubernetes.io/part-of: podtato-head
+        app.kubernetes.io/version: 0.1.1
+    spec:
+      containers:
+        - name: podtato-head-hat
+          image: podtato-head-hat:a
+```
+
+Now, let's assume that the configuration of that workload needs to be changed.
+In this example we assume that the image of that workload
+should be updated, but a configuration change is not limited to that.
+From here, you essentially have two options:
+
+* **Only update the configuration *without* updating the `app.kubernetes.io/version`
+label:** This can be useful when the change in the configuration should happen regardless
+of the result of any task or evaluation, e.g., when the previously used image has a critical vulnerability
+and the image must be updated as quickly as possible.
+To do that, change `podtato-head-frontend` as follows:
+
+```yaml
```

Comment:
please use external asset files and include them here for all the code examples in this file :)